h5p-bildetema/src/components/CollectionsController/CollectionController.tsx

```diff
@@ -36,13 +79,28 @@ const CollectionController = (): JSX.Element => {
     if (!collection) {
       return <CollectionsPage />;
     }
-    return <CollectionPage collectionTitle={collection} />;
+    return (
+      <CollectionPage
+        collectionTitle={collection}
+        showWrittenWords={showWrittenWords}
+        showArticles={showArticles}
+      />
+    );
   };
 
   return (
     <div className={`${styles.CollectionController} ${styles.mainSize}`}>
       <div className={styles.menuWrapper}>
-        <Breadcrumbs currentLanguageCode={langCode} breadCrumbs={breadCrumbs} />
+        <SubHeader
+          breadCrumbs={breadCrumbs}
```

Comment:
We might need to edit breadCrumbs to include showArticle and showWords search params when they are not the default value. To keep state in sync with the search params.

Reply:
Do you mean that we should remove the query from the url when we go to another page so that it reset back to the default state?

Reply:
We should try to keep the same value for showArticle and showWords on "temavisning" and "samlinger", so that the user will not not need to toggle these settings every time they change between these views. Currently they are taken back to the default value each time the user moves between "temavisining", search, and "samlinger".

Reply:
Fixed. Created global state with context. queryParams will only be part of the url on word view.